enterprise_subsidy/apps/subsidy/api.py

```diff
@@ -58,16 +65,14 @@ def create_subscription_subsidy(
     if kwargs.get('ledger'):
         return subsidy
 
-    from enterprise_access.apps.ledger.api import (
-        create_ledger,
-    )
     ledger = create_ledger(
         unit=unit,
         idempotency_key=create_idempotency_key_for_subsidy(subsidy),
     )
 
     subsidy.ledger = ledger
-    if kwargs.get('starting_balance'):
+    # The following condition is crafted to respect 0 as an explicit starting balance.
+    if 'starting_balance' in kwargs and kwargs['starting_balance'] is not None:
```

Comment:
similarly to the credit subsidy, i am going to suggest we make `starting_balance` an `arg` rather than a `kwarg`